scripts/minisat-2.2.1-export-parseSingleOption.patch

```diff
@@ -0,0 +1,76 @@
+From f394073844d6abe65dbd81eb6a787ae25e780e77 Mon Sep 17 00:00:00 2001
```

Comment:
Patch created from a git commit, better viewed here:
https://github.com/cmangla/minisat-2.2.1-for-cbmc/commit/f394073844d6abe65dbd81eb6a787ae25e780e77?diff=unified